parquet/src/arrow/arrow_writer/mod.rs

```diff
@@ -1297,6 +1343,12 @@ fn write_leaf(writer: &mut ColumnWriter<'_>, levels: &ArrayLevels) -> Result<usi
                         write_primitive(typed, array.values(), levels)
                     }
                 },
+                ArrowDataType::RunEndEncoded(_run_ends, _values) => {
+                    Err(ParquetError::NYI(
+                        "Int64ColumnWriter: Attempting to write an Arrow REE type that is not yet implemented"
```

Comment:
```suggestion
                        "Int64ColumnWriter: Attempting to write an Arrow RunEndEncoded type that is not yet implemented"
```
nit: IMO it would be easier for the end user to debug/understand the error if it does not use acronyms.

Reply:
Agreed!